homeassistant/components/paperless_ngx/coordinator.py

```diff
@@ -0,0 +1,132 @@
+"""Paperless-ngx Status coordinator."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import timedelta
+
+from pypaperless import Paperless
+from pypaperless.exceptions import (
+    InitializationError,
+    PaperlessConnectionError,
+    PaperlessForbiddenError,
+    PaperlessInactiveOrDeletedError,
+    PaperlessInvalidTokenError,
+)
+from pypaperless.models import Statistic
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError, ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER
+
+type PaperlessConfigEntry = ConfigEntry[PaperlessCoordinator]
+
+UPDATE_INTERVAL = 120
+
+
+@dataclass(kw_only=True)
+class PaperlessData:
+    """Describes Paperless-ngx sensor entity."""
+
+    statistic: Statistic | None = None
```

Comment:
As from the other conversation, multiple coordinators will be added in future PRs if needed, than this makes no sense. We don't need to wrap the return value in a dataclass. Instead we can return Statistic directly.
We can always add it back if needed in a future PR